cmd/yurt-controller-manager/app/core.go

```diff
@@ -53,3 +56,15 @@ func startNodeLifecycleController(ctx ControllerContext) (http.Handler, bool, er
 	go lifecycleController.Run(ctx.Stop)
 	return nil, true, nil
 }
+
+func startYurtHubCSRApproverController(ctx ControllerContext) (http.Handler, bool, error) {
+	clientSet := ctx.ClientBuilder.ClientOrDie("node-controller")
+	sharedInformerFactory := informers.NewSharedInformerFactory(clientSet, 10*time.Second)
```

Comment:
It's not need to new `sharedInformerFactory`, we can use `InformerFactory` in ctx parameter.

Reply:
The latest code has been commited.Using InformerFactory in ctx parameter.